.releaserc.yaml

```diff
@@ -1,7 +1,7 @@
 branch: master
 ci: false
-dryRun: true
 debug: false
+dryRun: false
```

Comment:
why do we need it with false? isn't this the default?

Reply:
Ooops, too late. I thought it will be easier to switch it in the future if needed.

Reply:
not big deal, was just curious

Reply:
we can keep such values turned to false, same for `debug` btw, so its easier to switch, as these two options will be most used during development/debugging